operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -103,6 +130,24 @@ Change the password of any existing user account using the HPE PDU web interface
 1. Use the **"admin"** menu (top right corner) to navigate to **"User Accounts"**.
 1. Click on the **"Edit"** icon (pencil) next to the user.
 1. Enter new password and make any other changes for that user account and click the **"Save"** button.
+1. If the `admin` password was changed, update Vault with the new credentials following procduere [Update Vault Credentials](#update-vault-credentials)
+
+### Update Vault Credentials
```

Comment:
This section needs to be ripped out or replaced. pdu-creds section of Vault is only for RTS/ServerTech PDUs.

Reply:
For HPE PDUs we expect the root user to exist and have redfish, so we can use the following to update the credentials in Vault
```
cray hsm inventory redfishEndpoints update XNAME --username root --password foobar
```

Reply:
removed